zeeguu/api/endpoints/user_activity_and_commitment.py

```diff
@@ -0,0 +1,109 @@
+import flask
+from flask import request
+
+from zeeguu.api.utils import json_result
+from . import api, db_session
+from zeeguu.api.utils.route_wrappers import cross_domain, requires_session
+from datetime import datetime, timedelta
+
+from zeeguu.core.model import User
+from zeeguu.core.model import UserCommitment
+import zeeguu.core.model
+
+from zeeguu.core.user_statistics.activity import activity_duration_by_day
+
+@api.route("/user_activity_and_commitment", methods=("GET",))
+@cross_domain
+@requires_session
+def user_activity_and_commitment():
+    """
+    Returns all the user activity and commitment data 
+    """
+    user = User.find_by_id(flask.g.user_id)
+    user_commitment = UserCommitment.query.filter_by(user_id=user.id).first()
+    user_activities = activity_duration_by_day(user)
+    return json_result({
+        "user_minutes": user_commitment.user_minutes,
```

Comment:
`user_minutes` is not informative enough. `committed_minutes` or `commitment_minutes` is better.